packages/dropdown-react/src/Dropdown.tsx

```diff
@@ -1,71 +1,114 @@
-import React from "react";
-import Downshift from "downshift";
+// @ts-ignore
+import CoreToggle from "@nrk/core-toggle/jsx";
+import React, { useState } from "react";
+import nanoid from "nanoid";
```

Comment:
We should make a task to replace all uuids with nanoid

Reply:
Done!